<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>649</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>641</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPaintViewImage" name="image_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>200</y>
     <width>121</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="length_button">
      <property name="text">
       <string>Length</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="box_button">
      <property name="text">
       <string>Box</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="circle_button">
      <property name="text">
       <string>Circle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="corner_button">
      <property name="text">
       <string>Corner</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="rotate_button">
      <property name="text">
       <string>Rotate</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="colormapBox">
      <item>
       <property name="text">
        <string>GRAY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AUTUMN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BONE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>COOL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HOT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HSV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JET</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OCEAN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PINK</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RAINBOW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SPRING</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SUMMER</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WINTER</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPaintViewImage</class>
   <extends>QLabel</extends>
   <header>qpaintviewimage.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
